downloads/models.py

```diff
@@ -146,6 +146,20 @@ def is_version_at_least_3_5(self):
     def is_version_at_least_3_9(self):
         return self.is_version_at_least((3, 9))
 
+    def clean(self):
+        super().clean()
+        if not re.match(r'^Python\s[\d.]+$', self.name):
+            raise ValidationError({
+                "name": "Release name must be in the format 'Python X.Y.Z' (e.g., 'Python 3.14.0')"
+            })
+
+    # def clean(self):
```

Comment:
is this necessary?

Reply:
ohh i copied from the issue i created 🤦🏼‍♂️